### ISSUE TITLE
Suggestion: Add uscourts.gov judicial milestone scraper

### ISSUE THREAD

**Issue by flooie** [freelawproject/courtlistener#1926]
[Judicial Milestones at US Courts.gov](https://www.uscourts.gov/judges-judgeships/judicial-milestones)

For example, a new judge Peter D Welte, was appointed and then made [Chief of North Dakota](https://www.uscourts.gov/judicial-milestones/peter-d-welte)

If we monitored the uscourts.gov judicial milestones we would be able to keep track of when our current judges change roles (elevated, retired etc.) as well as the appointment of magistrates and district court judges.  Im not necessarily arguing that it should automatically add new judges, but it could at the very least group expected new judges relatively easily.  

In the long run it would make adding new judges easier, and more organized and in theory could let us stay up-to-date.

**Comment by mlissner** [comment]
I'm not sure I understand what you mean by grouping expected new judges. Can you explain a bit more?

**Comment by flooie** [comment]
yeah ... sorry I mean ... I was suggesting that we tailor the errors to sentry to try to group errors around what was changing.  Tailor the error message to group death announcements together and group senior status together and group magistrates together in the error sent to sentry so it can be easily compiled.

Ideally though I think automating the whole process sounds best to me.

**Comment by mlissner** [comment]
Yeah, automating it sounds best to me too. I'm not sure we get very much by moving it off their website and into Sentry. Might as well write the content to a JSON log at that point....
